microbench/pom.xml

```diff
@@ -73,12 +73,12 @@
     <dependency>
       <groupId>org.openjdk.jmh</groupId>
       <artifactId>jmh-core</artifactId>
-      <version>0.9</version>
+      <version>1.7.1</version>
```

Comment:
Yes this benchmark was failing to run with the older version...time to update :)